DirectEthernet/src/main/pack/DirectEthernet.ini

```diff
@@ -1,6 +1,6 @@
 [info]
 name = Scala Direct Ethernet
-version = 1.0.0-M1
```

Comment:
I agree, though to do that we'd probably need to modify the build to produce the ini file from a template or something like that. I'll file an issue.